build.sbt

```diff
@@ -203,12 +203,12 @@ lazy val mimaSettings = Def.settings(
  * This returns the previous jar we released that is compatible with
  * the current.
  */
-val noBinaryCompatCheck = Set[String]("benchmark", "caliper", "generic", "spark")
+val noBinaryCompatCheck = Set[String]("benchmark", "caliper", "spark")
 
 def previousVersion(subProj: String) =
   Some(subProj)
     .filterNot(noBinaryCompatCheck.contains)
-    .map(s => "com.twitter" %% ("algebird-" + s) % "0.13.5")
+    .map(s => "com.twitter" %% ("algebird-" + s) % "0.13.7")
```

Comment:
we forgot to bump this a couple of times...

Reply:
nice catch! Now mima is complaining about the addition of the overrides to Double/FloatRing. It should be ok to just add 2 exclusion rules here.

Reply:
I think it might be possible to update this versions dynamically